src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -36,6 +37,11 @@ public class Hardware {
 			WRIST_ENCODER_PORT_A = 4,
 			WRIST_ENCODER_PORT_B = 5;
 
+	// Intake devices are ports 30-39
+	public static final int INTAKE_MOTOR_1 = 30;
+
+	public static final int INTAKE_LED_STRIP = 6, INTAKE_DISTANCE_SENSOR = 7, INTAKE_COLOR_SENSOR = 8;
```

Comment:
Should we still include `INTAKE_LED_STRIP` in here if the `ledStrip` method was removed?